autorag/evaluate/metric/retrieval.py

```diff
@@ -5,7 +5,13 @@
 def retrieval_metric(func):
     @functools.wraps(func)
     def wrapper(retrieval_gt: List[List[List[str]]], pred_ids: List[List[str]]) -> List[float]:
-        return list(map(lambda x: func(x[0], x[1]), zip(retrieval_gt, pred_ids)))
+        results = []
+        for gt, pred in zip(retrieval_gt, pred_ids):
+            if gt == [[]]:
+                results.append(None)
```

Comment:
Inserting None when there is no gt in there. Good idea!

Because pd.mean has `skipna` option and it is True as default.